frontend/src/APIClients/mutations/StoryMutations.ts

```diff
@@ -199,4 +199,20 @@ export const SOFT_DELETE_STORY = gql`
     }
```

Comment:
can you revert the changes in this file? uploading files via frontend likely wont work like this (jenn L is working on this for uploading resumes) 
generally also should avoid merging stuff we cant/havent tested